src/battle-tooltips.ts

```diff
@@ -1526,6 +1526,13 @@ class BattleTooltips {
 			else basePower = 20;
 			value.set(basePower);
 		}
+		if (move.id === 'furycutter'){
+			// what will actually retrieve the multiplier?
```

Comment:
You have to keep track of the multiplier yourself. The server data isn't available to clients